### ISSUE TITLE
CRAN: `R_ExternalPtrTag: argument of type NILSXP is not an external pointer` from mirai package

### ISSUE THREAD

**Issue by HenrikBengtsson** [HenrikBengtsson/future.mirai#14]
CRAN just reported on lots of:

```
Error in request(.context(envir[["sock"]]), data = data, send_mode = 3L,  : 
  R_ExternalPtrTag: argument of type NILSXP is not an external pointer
Calls: plan ... run.MiraiFuture -> do.call -> <Anonymous> -> request
Execution halted
```

for several R-devel platforms;

* https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-gcc/future.mirai-00check.html
* https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-fedora-gcc/future.mirai-00check.html
* https://www.r-project.org/nosvn/R.check/r-devel-windows-x86_64/future.mirai-00check.html

# See also

It looks like **mirai** itself has this problem too, cf. <https://cran.r-project.org/web/checks/check_results_mirai.html>

**Comment by shikokuchuo** [comment]
These externalptr issues have now all been cured. It seems you still have one outstanding check error - looks like it's that unreliable test I think you've already fixed.

**Comment by HenrikBengtsson** [comment]
Fixed in **nanonext** 1.1.1, cf. https://github.com/shikokuchuo/nanonext/issues/35.

**Comment by HenrikBengtsson** [comment]
Thanks.

> These externalptr issues have now all been cured. It seems you still have one outstanding check error - looks like it's that unreliable test I think you've already fixed.

Yes. I'll give another go at that sporadic failure, cf. https://github.com/HenrikBengtsson/future.mirai/issues/15.
